### DISCUSSION TITLE
How can I configure Bundlemon to recognize hashes generated by Rollup 4?

### DISCUSSION THREAD

**Discussion by goibon** [LironEr/bundlemon#191] [Q&A] [answered]
I'm using Bundlemon to keep track of my chunk sizes in a project that uses Vite which in turn uses Rollup to bundle. My chunks follow a naming convention of `[name]-[hash].js` which used to mean that they would be named something like `chunkNameA-5fdd3f1d.js`. After upgrading to Vite 5 which in turn upgraded Rollup from v3 to v4 the hashes are now different ([as mentioned as a breaking change in Rollup release notes](https://github.com/rollup/rollup/releases/tag/v4.0.0)). The new hashes are no longer just hex characters but seem to include `a-z`, `0-9`, `_` and `-`. The use of `-` in hashes seems to cause some issues with file matching with bundlemon.
My bundlemon config uses a `path` setting that looks like: `"path": "assets/*-<hash>.js",` which ended up showing part of the hash in the bunldemon output, eg. `assets/chunkNameA-Zxa-(hash).js` where `Zxa-` is part of the hash but not recognized as such.
My initial first attempt at addressing this was to use a new chunk file naming in rollup `[name].[hash].js` and a new `path` in bundlemon: `"path": "assets/*.<hash>.js",` but this also does not properly match the files so if a file has a `-` in the hash then bundlemon reports it as being removed.
I realize that potentially I could also resolve this by finding a way of getting vite/rollup to output compatible hashes again but my initial attempts at figuring out how were unfruitful so I thought I'd ask here.

Is there any way I can get Bundlemon to properly match the new hashes?

**Comment by LironEr** [comment, marked as answer]
Released in [v2.1.0](https://github.com/LironEr/bundlemon/releases/tag/bundlemon%402.1.0)
